client/my-sites/upgrades/cart/cart-item.jsx

```diff
@@ -85,6 +95,41 @@ module.exports = React.createClass( {
 		return info;
 	},
 
+	handleDomainVolumeSelection: function( event ) {
+		event.preventDefault();
+		let volume = parseInt( event.target.value );
```

Comment:
also pretty important here is that we should always include our radix when using `parseInt` (at least according to Mozilla and others)

```js
const volume = parseInt( event.target.value, 10 );
```